src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.mdx

```diff
@@ -0,0 +1,12 @@
+---
+id: sharedUX/Components/AppLink
+slug: /shared-ux/components/redirect-app-link
+title: Redirect App Link
+summary: The component for redirect links.
+tags: ['shared-ux', 'component']
+date: 2022-02-01
+---
+
+> This documentation is in progress.
+
+This component has been refactored. Instead of requiring the entire application, it instead takes just navigateToUrl and currentAppId$. This makes the component more lightweight.
```

Comment:
```suggestion
**This component has been refactored.** Instead of requiring the entire `application`, it instead takes just `navigateToUrl` and `currentAppId$`. This makes the component more lightweight.
```